requirements/devel.txt

```diff
@@ -4,6 +4,9 @@
 # add the testing dependencies
 -r test.txt
 
+# add integration dependencies
+-r integrate.txt
+
```

Comment:
```suggestion
```
lets d not do this as PL ould reinstall also TM and not sure what the order is...